README.md

```diff
@@ -117,7 +117,7 @@ The following is a list of other *OPTIONAL* props you may provide to the `DateRa
 // input related props
 startDatePlaceholderText: PropTypes.string,
 endDatePlaceholderText: PropTypes.string,
-disabled: PropTypes.bool,
+disabled: PropTypes.oneOfType([PropTypes.bool, PropTypes.oneOf([START_DATE, END_DATE])]),
```

Comment:
We could probably pull this out into its own variable

Reply:
Should I reference it from "DateRangePickerShape.js" in the three other locations it is used, and leave it as it is (or simplify it) here in the readme?

Reply:
I cleaned it up! :P But left it in the README for clarity.